server/app/controllers/dev/DatabaseSeedController.java

```diff
@@ -133,11 +120,14 @@ private QuestionDefinition insertAddressQuestionDefinition() {
     return questionService
         .create(
             new AddressQuestionDefinition(
-                "address",
-                Optional.empty(),
```

Comment:
Here and elsewhere, `Optional.empty()` refers to the `enumeratorId`, which is set to empty by the `AutoValue` and thus not set on the right side of the diff.